prow/cmd/deck/main.go

```diff
@@ -190,11 +190,12 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.BoolVar(&o.gcsCookieAuth, "gcs-cookie-auth", false, "Use storage.cloud.google.com instead of signed URLs")
 	fs.BoolVar(&o.rerunCreatesJob, "rerun-creates-job", false, "Change the re-run option in Deck to actually create the job. **WARNING:** Only use this with non-public deck instances, otherwise strangers can DOS your Prow instance")
 	fs.BoolVar(&o.allowInsecure, "allow-insecure", false, "Allows insecure requests for CSRF and GitHub oauth.")
-	fs.IntVar(&o.timeoutListingProwJobs, "timeout-listing-prowjobs", 30, "Timeout for listing prowjobs in seconds.")
 	fs.BoolVar(&o.dryRun, "dry-run", false, "Whether or not to make mutating API calls to GitHub.")
 	fs.Var(&o.tenantIDs, "tenant-id", "The tenantID(s) used by the ProwJobs that should be displayed by this instance of Deck. This flag can be repeated.")
 	o.config.AddFlags(fs)
 	o.instrumentation.AddFlags(fs)
+	o.controllerManager.TimeoutListingProwJobsDefault = 30
```

Comment:
This actually sets the duration to 30 nanoseconds I think (or something else very small). This is later adjusted for on line 377 by multiplying by `time.Second`, but it would better to set the value correctly here by setting it to `30*time.Second`. This is how we get benefit from using a duration variable rather than fixing the units to seconds and using an int var. Please fix this in all locations.

aside: I'm surprised there isn't a compiler or linter warning for setting a duration variable to a literal without including a `time.*` constant multiplier.